custom-tsdb/README.md

```diff
@@ -54,3 +54,85 @@ Long-term, Kubecost recommends storing the configuration in `cloud-integration.j
 - [AWS](../aws/)
 - [Azure](../azure/)
 - [GCP](../gcp/)
+
+## List of Metrics
+
```

Comment:
I think showing the query used to obtain this list would be super helpful!

```
group by(__name__) ({__name__!="", job="kubecost-byo-prom-cost-analyzer"})
```

Where `job` is the name of the scrapeconfig you have configured for Prometheus